src/overview/structure.md

```diff
@@ -18,19 +26,25 @@ yourproject/
     services.yaml
     routes.yaml
   .platform.app.yaml
-  <your files here>
+  <your application code>
```

Comment:
Let's go with "your application files", since it's technically not all code, and code implies inlining it which would of course be silly.

Reply:
I think this was a case of brain and hands not agreeing on what to type, since "your application code" makes no sense.

Reply:
Fixed.